### PR TITLE
Update package.json for upcoming polymer-analyzer rename

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
     "type": "git",
-    "url": "https://github.com/Polymer/hydrolysis.git"
+    "url": "https://github.com/Polymer/polymer-analyzer.git"
   },
   "main": "index.js",
   "files": [
-    "hydrolysis-analyzer.html",
-    "hydrolysis.html",
-    "hydrolysis.js",
     "index.js",
```
Comment (rictic) [inline package.json]: This is now `lib/index.js` so it can be removed.

Comment (rictic) [inline package.json]: Should add custom_typings/ to this array though, so downstream installers get all the definitions they need.

**Comment by rictic** [comment]
LGTM once the above two comments are addressed.

File: package.json (hunk 1 above)
Comment (justinfagnani) [inline package.json]: done and done

**Pull request merged**